### PR TITLE
Fix documentation links

### LINKED ISSUE #983: Broken link for Kafka input
Version 1.0.0
System/ Nodes/ installed, available input types:

https://www.graylog.org/documentation/sending/kafka/ Should redirect to Kafka input on documentation page but Kafka input is missing in documentation.

### PR DESCRIPTION
**Pull request by bernd**
* Removes links to nonexistent Kafka documentation. (see #983 and Graylog2/documentation#19)
* Introduces a `DocsHelper` enum to avoid hardcoding links in the code base. (see Graylog2/graylog2-web-interface#1466)

This is targeted at the master branch. The 68ea4a8 should be cherry-picked into the 1.1 branch to fix #983.

### CODE REVIEW

File: graylog2-plugin-interfaces/src/main/java/org/graylog2/plugin/DocsHelper.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+    DocsHelper(String path) {
+        this.path = path;
+    }
+
+    @Override
+    public String toString() {
+        final String version = Version.CURRENT_CLASSPATH.major + "." + Version.CURRENT_CLASSPATH.minor;
+
+        final StringBuffer sb = new StringBuffer(DOCS_URL)
```
Comment (joschi) [inline graylog2-plugin-interfaces/src/main/java/org/graylog2/plugin/DocsHelper.java]: The use of `StringBuffer` is unnecessary here and could be replace by simple string concatenation. Additionally `StringBuffer` adds a "tax" for being thread safe and needing to synchronize on write operations like `append()`.

File: graylog2-plugin-interfaces/src/main/java/org/graylog2/plugin/DocsHelper.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+                .append(version)
+                .append("/pages/")
+                .append(path);
+
+        return sb.toString();
+    }
+
+    public String toLink(String title) {
+        final StringBuffer sb = new StringBuffer("<a href=\"")
```
Comment (joschi) [inline graylog2-plugin-interfaces/src/main/java/org/graylog2/plugin/DocsHelper.java]: See above.

File: graylog2-plugin-interfaces/src/main/java/org/graylog2/plugin/DocsHelper.java [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+ * THE SOFTWARE.
+ */
+
+package org.graylog2.plugin;
+
+public enum DocsHelper {
+    PAGE_SENDING_JSONPATH("sending_data.html#json-path-from-http-api-input"),
+    PAGE_ES_CONFIGURATION("configuring_es.html");
+
+    public static String DOCS_URL = "http://docs.graylog.org/en/";
```
Comment (joschi) [inline graylog2-plugin-interfaces/src/main/java/org/graylog2/plugin/DocsHelper.java]: Missing `final` keyword.

File: graylog2-plugin-interfaces/src/main/java/org/graylog2/plugin/DocsHelper.java [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    public static final String HELP_DOCS = "http://docs.graylog.org/";
```
Comment (joschi) [inline graylog2-plugin-interfaces/src/main/java/org/graylog2/plugin/DocsHelper.java]: I wouldn't move those three constants into the `DocHelper` class as they are only used in the `UI` class and are kind of different from the other documentation links that are being constructed in `DocHelper`.

**Comment by bernd** [comment]
Updated after comments.

**Comment by joschi** [comment]
LGTM. :+1:

**Pull request merged**